romana-install/group_vars/vagrant

```diff
@@ -1,7 +1,27 @@
 ram_size:
   controller:
     devstack: 4096
-    kubernetes: 1024
+    kubernetes: 2048
   compute:
     devstack: 1024
```

Comment:
The k8s sizing was a reaction to the recent demo when it seemed to be out of RAM.